package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "react-semantic-toasts",
-    "version": "0.3.6",
+    "version": "0.3.7",
```

Comment:
I prefer that you leave the version bumping up to me.

Reply:
Ok I'll do it